bin/build-blocks.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+/* eslint no-console: 0 */
+
+const fs = require( 'fs' );
+const path = require( 'path' );
+const execSync = require( 'child_process' ).execSync;
+
+const blocks = [
+	'hello-dolly/hello-block.js',
+	// 'jetpack/tiled-gallery/tiled-gallery.jsx',
+];
+
+const outputDir = path.resolve( '_inc/build/blocks' );
```

Comment:
This directory makes sense to me, but I'd like to confirm with someone from @Automattic/jetpack-crew that this is the right directory we'll want to place the externally built blocks.

Reply:
True; it probably isn't the right place. We'll need to wait for convos with @dereksmart and @oskosk